tasks/synopsys-detect-task/ts/DetectADOConstants.ts

```diff
@@ -0,0 +1,17 @@
+export abstract class DetectADOConstants {
+    static readonly BLACKDUCK_ID = "BlackDuckService"
+    static readonly BLACKDUCK_PROXY_ID = "BlackDuckProxyService"
+
+    static readonly BLACKDUCK_API_TOKEN = "apitoken"
+    static readonly BLACKDUCK_USERNAME = "username"
+    static readonly BLACKDUCK_PASSWORD = "password"
+
+    static readonly PROXY_USERNAME = "username"
+    static readonly PROXY_PASSWORD = "password"
+
+    static readonly DETECT_FOLDER = "DetectFolder"
+    static readonly DETECT_VERSION = "DetectVersion"
+    static readonly DETECT_ARGUMENTS = "DetectArguments"
+
+    static readonly ADD_TASK_SUMMARY = "AddTaskSummary"
```

Comment:
Maybe I'm picky, but having a prefix/suffix to make it clear that these represent field names rather than field values would be helpful for future developers.

Reply:
These match the values in the json files used to setup the tasks. I'm concerned changing them may cause issues for users that already have this extension setup